### ISSUE TITLE
Bitbucket support

### ISSUE THREAD

**Issue by samrm111** [inch-ci/inch_ci-web#115]
Any planned support for Bitbucket?

**Comment by rrrene** [comment]
Yes, but not in the near future. There are a lot of things on the roadmap for late 2016 and BB support is one thing I would like to do.

But I can't make any promises regarding a definite launch date. :cry:
